pkg/sfu/session.go

```diff
@@ -210,6 +209,15 @@ func (s *Session) setRelayedDatachannel(peerID string, datachannel *webrtc.DataC
 	})
 }
 
+func (s *Session) BroadcastAPIChannelMessage(msg ChannelAPIMessage, origin string) {
+	peers := s.Peers()
```

Comment:
Can we make this a generic function? that accepts msg and channel?

Reply:
Hey @OrlandoCo I addressed some of your comments from chat, making peer.SendAPIChannelMessage generic and preventing marshaling for every peer.